DocumentsFromSnapshotMigration/README.md

```diff
@@ -71,6 +71,7 @@ To see the default shard size, use the `--help` CLI option:
 | --target-aws-region               | The AWS region for the target cluster. Required if using SigV4 authentication                                                                            |
 | --target-aws-service-signing-name | The AWS service signing name (e.g. 'es' for Amazon OpenSearch Service, 'aoss' for Amazon OpenSearch Serverless). Required if using SigV4 authentication  |
 | --target-insecure                 | Flag to allow untrusted SSL certificates for target cluster                                                                                              |
+| --target-compression              | Flag to enable request compression for target cluster                                                                                                    |
 | --documents-per-bulk-request      | Optional. The number of documents to be included within each bulk request sent. Default: no max (controlled by documents size)                           |
 | --documents-size-per-bulk-request | Optional. The maximum aggregate document size to be used in bulk requests in bytes. Default: 10 MiB                                                      |
-| --max-connections                 | Optional. The maximum number of connections to simultaneously used to communicate to the target. Default: 20                                             |
+| --max-connections                 | Optional. The maximum number of connections to simultaneously used to communicate to the target. Default: 10                                             |
```

Comment:
why did this value change?

Reply:
Changed outside of this PR, was reconciling the actual value with the docs here.

When I had been testing with 4GB workers and 10MB requests on ECS, I got better throughput with 10 connections than 20.

We can and should do further testing to identify good starting out values for this.